app/views/shared/_error.html.slim

```diff
@@ -0,0 +1,9 @@
+- if model.errors.any?
+  #error_explanation
+    h2
+      = pluralize(model.errors.count, "error")
+      |  prohibited this #{model.class} from being saved:
+    ul
+      - model.errors.full_messages.each do |msg|
+        li
+          = msg
```

Comment:
https://getbootstrap.com/docs/4.5/components/alerts/
style this with bootstrap please for alerts.
since its an error it should use the red one